packages/node/src/index.ts

```diff
@@ -99,7 +100,21 @@ async function downloadInstallAndBundle({
     meta
   );
   const spawnOpts = getSpawnOptions(meta, nodeVersion);
-  await runNpmInstall(entrypointFsDirname, [], spawnOpts, meta, nodeVersion);
+
+  if (config.zeroConfig) {
```

Comment:
```suggestion
  if (config.zeroConfig && config.installCommand === 'string') {
```

This might fix it. Also I think we should turn this into a helper function so all builders can use it. Perhaps pass config to `runNpmInstall()`

Reply:
Wow, that worked. I did a little cleanup. Dunno about passing `config` to `runNpmInstall()`. I would prefer to not touch public APIs in this PR.